src/lib/constants/element/datepicker.py

```diff
@@ -0,0 +1 @@
+SELECTOR = "a.ui-state-default"
```

Comment:
That would then be in a module named elements which would live in the lib package. In time hundreds of (sub) classes would then be in one module... Also then there would have to be an elements module in the constants package where the constants would have to be uniquely named (and the dev would have to guess which element that is on the page based on the name inference) ... 
Current design avoids both issues by explicitly exposing the page hierarchy without classes (since classes have no place in the constants package). 
Taking that into consideration, would you still define elements as classes in constants.elements?